### PR TITLE
Add initialize flag to v1/shell

### PR DESCRIPTION
**Pull request by jakecoll**

### CODE REVIEW

**Review by diurnalist — APPROVED** [review]
LGTM

File: heatclient/v1/shell.py [hunk 1]
```diff
@@ -69,6 +69,8 @@ def show_deprecated(deprecated, recommended):
 @utils.arg('-t', '--timeout', metavar='<TIMEOUT>',
            type=int,
            help=_('Stack creation timeout in minutes.'))
+@utils.arg('-i', '--initialize', default=False, action="store_true",
+           help=_('Only initialize stack and do not allocate resrouces.'))
```
Comment (priteau) [inline heatclient/v1/shell.py]: s/resrouces/resources/

**Review by priteau — CHANGES_REQUESTED** [review]

File: heatclient/v1/shell.py (hunk 1 above)
Comment (jakecoll) [reply heatclient/v1/shell.py]: fixed type

Comment (jakecoll) [reply heatclient/v1/shell.py]: *typo

**Pull request merged**